misk-feature-testing/src/main/kotlin/misk/feature/testing/TestFeatureFlags.kt

```diff
@@ -0,0 +1,68 @@
+package misk.feature.testing
+
+import com.google.common.util.concurrent.AbstractIdleService
+import misk.feature.Attrs
+import misk.feature.Feature
+import misk.feature.FeatureFlags
+import misk.feature.FeatureService
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * In-memory test implementation of [FeatureFlags] that allows flags to be overridden.
+ */
+@Singleton
+class TestFeatureFlags @Inject constructor() : AbstractIdleService(),
+    FeatureFlags,
+    FeatureService {
+  override fun startUp() {}
+  override fun shutDown() {}
+
+  private val overrides = mutableMapOf<Feature, Any>()
+
+  override fun getBool(feature: Feature, token: String, attrs: Attrs?): Boolean {
```

Comment:
Bool –> Boolean

Reply:
Should I make it `GetInteger` then?

Reply:
No, a good rule is to never invent synonyms. Kotlin gives us Int and Boolean and those are the words we should use.